src/@dvcorg/gatsby-theme-iterative/components/HamburgerMenu/styles.module.css

```diff
@@ -135,7 +135,7 @@
 .sectionHeading {
   display: block;
   padding: 15px 0;
-  font-family: var(--font-brandon);
+  font-family: var(--font-base);
```

Comment:
`--font-brandon` is now `--font-base` to be more generic in the theme, which trickles down here in its implementation.